foodhub_product_fields/models/product_template.py

```diff
@@ -1,16 +1,24 @@
-# SPDX-FileCopyrightText: 2022 Coop IT Easy SC
-#
-# SPDX-License-Identifier: AGPL-3.0-or-later
-
 from odoo import fields, models
 
 
 class ProductTemplate(models.Model):
     _inherit = "product.template"
 
-    # All below fields just have copy=True added to them.
-    volume = fields.Float(copy=True)
-    weight = fields.Float(copy=True)
+    # All below fields have required=True added to them.
+    description_sale = fields.Text(required="True")
+    public_categ_ids = fields.Many2many(required="True")
+    pos_categ_id = fields.Many2one(required="True")
+    list_price = fields.Float(required="True")
+
+    # The below field have copy=True added to them.
+    origin_country_id = fields.Many2one(
+        "res.country",
```

Comment:
Apparently the module name here can be removed.

Reply:
Hmm, what module name ?? ^^

Reply:
Sorry, model name. res.country.